CHANGELOG.md

```diff
@@ -2,30 +2,31 @@
 
 ### Enhancements
 
-* Added an additional trace logger for NLP debugging.
+- Added an additional trace logger for NLP debugging.
```

Comment:
doc nit, looks like the .md changes were auto changed by an editor? please revert.

Reply:
Fixed, thanks